src/components/Settings/index.tsx

```diff
@@ -10,20 +10,29 @@ import useDisableScrolling from 'hooks/useDisableScrolling'
 import { useOnClickOutside } from 'hooks/useOnClickOutside'
 import { Portal } from 'nft/components/common/Portal'
 import { useIsMobile } from 'nft/hooks'
-import { useMemo, useRef } from 'react'
-import { useModalIsOpen, useToggleSettingsMenu } from 'state/application/hooks'
+import { useCallback, useMemo, useRef } from 'react'
+import { X } from 'react-feather'
+import { useCloseModal, useModalIsOpen, useToggleSettingsMenu } from 'state/application/hooks'
 import { ApplicationModal } from 'state/application/reducer'
 import { InterfaceTrade } from 'state/routing/types'
 import { isUniswapXTrade } from 'state/routing/utils'
 import styled from 'styled-components'
-import { CloseIcon, Divider, ThemedText } from 'theme'
+import { Divider, ThemedText } from 'theme'
 import { Z_INDEX } from 'theme/zIndex'
 
 import MaxSlippageSettings from './MaxSlippageSettings'
 import MenuButton from './MenuButton'
 import RouterPreferenceSettings from './RouterPreferenceSettings'
 import TransactionDeadlineSettings from './TransactionDeadlineSettings'
 
+const CloseButton = styled.button`
```

Comment:
Close Button is no longer aligned vertically with Header Title
![Screenshot 2023-08-24 at 10 12 52 AM](https://github.com/Uniswap/interface/assets/11512321/d8025d2e-911e-47ed-ac04-9a4c9f9228ef)

Reply:
Are you sure? the button on the left is 24px wide, and the `Settings` text wrapper has a 24px padding on the right.

I've got a ruler up on the screen and it stays in the same spot when I flip between prod and this vercel build.

Reply:
charlie is right i am observing the same thing and its different than whats on prod

Reply:
idk how I'm not seeing this

I'll check other browsers

Reply:
oh thats a good point - we both use firefox lol

Reply:
oh lmao I misread--I thought @cbachmeier was saying it's not centered in the screen

ok I see it, will fix.

Reply:
fixed